src/main/resources/db/migration/V20231003__Create_initial_entity_tables.sql

```diff
@@ -156,7 +156,7 @@ CREATE TABLE link_person_instances
   id          BIGSERIAL PRIMARY KEY,
   person_id   BIGINT                                    NOT NULL,
   instance_id BIGINT                                    NOT NULL,
-  created_at  TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP(3) NOT NULL
+  created_at  TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP(3) NULL
```

Comment:
Created at should not be nullable

Reply:
sadly the DEFAULT goes AFTER the check of NULL in the ORM. But yeah then we just need to make the required to false in the orm.

Reply:
nvm i had a brainfart somehow there. But it is right that the @UpdatedTimestamp does some weird sh*t and it needs to be required=False in the ORM. But in the sql it can stay.